### DISCUSSION TITLE
Remove "Stand with Ukraine" bottom banner?

### DISCUSSION THREAD

**Discussion by n-pn** [sveltejs/kit#7527] [General]
Semi-related article: https://poignardazur.github.io/2022/11/05/political-messages-in-the-rust-blog/

I think political issues should be avoided or at least makes it more subtle so everyone can tolerate them.

Coming from a country that was bombed to Stone Age by US in the past (my grand father was killed in that war, too), to be honest I feel that you westerners' reactions for this situation is hypocrisy as best. You didn't do anything when "some" country bombed the shit out of Syria, Iraq, Afghanistan, so why react now? Keep in mind that US is still actively use drone attack on Syria, with no plan on stopping. Let's not forget the infamous Time magazine cover that blatantly declared that bombing civilians was justified, either.

Frankly the banner bugged me so bad that I've stopped visiting the website since then, and only visit it just recently for the new breaking changes.  Should I just take it as a hint that Svelte community does not welcome people they deem as lesser humans?

Honestly I don't even want to make this post, because I'm sure the feedback won't be anything pleasant, but as my project (written in SvelteKit) continues to grow and depends more and more on Svelte this is a risk factor I have to consider seriously. 

P/S: I'm fine with the Ukrainian flag on the top left banner, but put a sticky footer that takes a lot of space with no option to hide is harassing.

**Comment by mustofa-id** [comment]
I use browser extensions called [uBlock Origin](https://github.com/gorhill/uBlock) to block elements on a site that I don't like.
